smpe/bld/service/ptf-bucket.txt

```diff
@@ -0,0 +1 @@
+UO00001 -
```

Comment:
The official ptf-bucket.txt that goes into staging must be created by Nayer, as it **MUST** hold PTF numbers maintained by RETAIN. So the file must be removed from this pull request. Nayer & I are currently waiting on an external factor to start the creation of ptf-bucket,txt, but I hope to have this soon.